lms/static/sass/_overrides.scss

```diff
@@ -12,6 +13,63 @@
     }
   }
 
+
+  // header buttons
```

Comment:
Buttons like "Help" next to the profile picture are aligned vertically in the purple header, but logging out I see that Register / Sign-In really aren't. Is there any way to fix this?